app/src/main/java/com/academy/android/ui/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,185 @@
+package com.academy.android.ui.profile
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.academy.android.R
+import com.skydoves.landscapist.glide.GlideImage
+
+@Composable
+fun ProfileScreen(vm: ProfileViewModel) {
+    val profileData: ProfileInfo by vm.profileData.collectAsState(initial = ProfileInfo())
+    val isInEditMode: Boolean by vm.isInEditMode.collectAsState(initial = false)
+
+    val marginStandard = dimensionResource(id = R.dimen.spacing_16)
+    val marginDouble = dimensionResource(id = R.dimen.spacing_32)
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        TopControls(
+            marginStandard = marginStandard,
+            marginDouble = marginDouble,
+            vm = vm,
+            isInEditMode = isInEditMode
+        )
+        Avatar(profPic = profileData.profPic)
+        ProfileItemList(
+            marginStandard = marginStandard,
+            marginDouble = marginDouble,
+            profileItemList = profileData.profileItemList,
+            isInEditMode = isInEditMode,
+            isChangesDiscarded = profileData.isChangesDiscarded
+        )
+    }
+}
+
+@Composable
+private fun TopControls(
+    marginStandard: Dp,
+    marginDouble: Dp,
+    vm: ProfileViewModel,
+    isInEditMode: Boolean
+) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(
+                top = marginStandard,
+                start = marginStandard,
+                end = marginStandard,
+                bottom = marginDouble
+            ),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        IconButton(onClick = {
+            if (isInEditMode) {
+                vm.discardChanges()
```

Comment:
and need to clear focus for current field

Reply:
Done.     
Though, [another issue](https://github.com/mik629/AndroidAcademyClient/issues/20) exists